eras/conway/impl/src/Cardano/Ledger/Conway/Genesis.hs

```diff
@@ -19,17 +20,20 @@ import Cardano.Ledger.Binary (
  )
 import Cardano.Ledger.Binary.Coders
 import Cardano.Ledger.Conway.Era (ConwayEra)
+import Cardano.Ledger.Conway.PParams (UpgradeConwayPParams)
 import Cardano.Ledger.Core
 import Cardano.Ledger.Crypto (Crypto)
 import Cardano.Ledger.Keys (GenDelegs (..))
 import Data.Aeson (FromJSON (..), ToJSON, object, withObject, (.:))
 import Data.Aeson.Types (KeyValue (..), ToJSON (..))
+import Data.Functor.Identity (Identity)
 import Data.Unit.Strict (forceElemsToWHNF)
 import GHC.Generics (Generic)
 import NoThunks.Class (NoThunks)
 
-newtype ConwayGenesis c = ConwayGenesis
+data ConwayGenesis c = ConwayGenesis
   { cgGenDelegs :: GenDelegs c
```

Comment:
GenDelgs actually need to be removed. This was the initial plan: replace genesis keys. But now since conway implements governance we no longer need genesis keys